deployment_scripts/linode-marketplace-jupyterlab/jupyterlab-deploy.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+set -e
+trap "cleanup $? $LINENO" EXIT
+
+## JupyterLab Settings
+#<UDF name="soa_email_address" label="Email address (for the Let's Encrypt SSL certificate)" example="user@domain.tld">
+
+## Linode/SSH Security Settings
+#<UDF name="username" label="The limited sudo user to be created for the Linode">
+#<UDF name="pubkey" label="The SSH Public Key that will be used to access the Linode" default="">
+#<UDF name="disable_root" label="Disable root access over SSH?" oneOf="Yes,No" default="No">
+
+## Domain Settings
+#<UDF name="token_password" label="Your Linode API token. This is needed to create your server's DNS records" default="">
+#<UDF name="subdomain" label="Subdomain" example="The subdomain for the DNS record: www (Requires Domain)" default="">
+#<UDF name="domain" label="Domain" example="The domain for the DNS record: example.com (Requires API token)" default="">
+
+# git repo
+export GIT_REPO="https://github.com/ericruzanski/marketplace-apps.git"
```

Comment:
```suggestion
export GIT_REPO="https://github.com/akamai-compute-marketplace/marketplace-apps.git"
```

Reply:
Will flip this back once PR changes pass test deployment.